spec/dataframe_spec.rb

```diff
@@ -561,6 +570,13 @@
 
         expect(@df_mi).to eq(answer)
       end
+
+      it "assigns correct name given empty dataframe" do
+        df_empty = Daru::DataFrame.new([], index: @multi_index, order: @order_mi)
+        df_empty[:c, :one, :bar] = 1..12
+
+        expect(df_empty[:c, :one, :bar].name).to eq "conebar"
+      end
     end
```

Comment:
Ah makes sense. I had forgotten that. My bad.

But it's good to have a check anyway.